src/components/Select/Select.tsx

```diff
@@ -241,6 +244,7 @@ export const Select = React.forwardRef<HTMLButtonElement, SelectProps>(function
             },
             [handleClose, onBlur],
         ),
+        isDisabled: mobile,
```

Comment:
Lets remove this line and don't call `handleClose` in `onBlurWithin` in case of `mobile` device